libs/awx/config.jsonnet

```diff
@@ -5,24 +5,16 @@ config.new(
   name='awx-operator',
   specs=[
     {
-      output: 'v0.25.0',
-      localName: 'awx',
-      prefix: '^com\\.ansible\\.awx\\..*',
-      crds: ['https://raw.githubusercontent.com/ansible/awx-operator/0.25.0/config/crd/bases/awx.ansible.com_awxs.yaml'],
-    },
-    {
-      output: 'v0.30.0',
+      output: 'v1.0.0',
       localName: 'awx',
       prefix: '^com\\.ansible\\.awx\\..*',
-      crds: ['https://raw.githubusercontent.com/ansible/awx-operator/0.30.0/config/crd/bases/awx.ansible.com_awxs.yaml'],
+      crds: ['https://raw.githubusercontent.com/ansible/awx-operator/1.0.0/config/crd/bases/awx.ansible.com_awxs.yaml'],
     },
     {
-      output: 'v1.0.0',
```

Comment:
shouldn't we leave the existing blocks otherwise will not be able to render in the meantime. We can add this block below `0.25.0` and `0.30.0`

```
    {
      output: 'v1.0.0',
      localName: 'awx',
      prefix: '^com\\.ansible\\.awx\\..*',
      crds: ['https://raw.githubusercontent.com/ansible/awx-operator/1.0.0/config/crd/bases/awx.ansible.com_awxs.yaml'],
    },
    {
      output: 'v1.1.4',
      localName: 'awx',
      prefix: '^com\\.ansible\\.awx\\..*',
      crds: ['https://raw.githubusercontent.com/ansible/awx-operator/1.1.4/config/crd/bases/awx.ansible.com_awxs.yaml'],
    }
```

Reply:
you could also put this in a `for loop` see https://github.com/influxdata/k8s-jsonnet-libs/blob/2fbf02f1ff43316ce5087f0da6e120501b5a70fe/libs/cert-manager/config.jsonnet#L2-L5

Reply:
0.30.0 and 0.25.0 are old versions that should longer be used, nothing is currently referencing them - interesting idea with the loop